### PR TITLE
Define Goals and non Goals of the project

### PR DESCRIPTION
**Pull request by aojea**
clarify the project scope

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,29 @@ See [TESTING](docs/testing/README.md)
...
+Covered Work: e2e testing, scalability, performance and reliability
+
+P0: Provide Early Feedback
+
+Stakeholders: SIG Network, Network Policy API Working Group
+Covered: New features implemented under feature gates for fast iteraton
+
+### Non Goals
+
+Implementing additional functionalities unrelated to Network Policies
```
Comment (danwinship) [inline README.md:87]: "Implementing functionality that is not part of an official or proposed Kubernetes network policy API" ?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,29 @@ See [TESTING](docs/testing/README.md)
...
+
```
Comment (danwinship) [inline README.md:88]: "Supporting production use" should be either a Goal or a Non-Goal. (ie, how do we handle bug reports?)

Likewise, "supporting use at scale"

Comment (aojea) [reply README.md:88]: I'm trying to indicate that people is free to use it as they want and feedback is really appreciated:  "given enough eyeballs, all bugs are shallow", but this is an open source project with best effort support ...  the main goal of the project is zero bugs ... best performance ... same as we do with KIND ... don't know how to write that

File: README.md (hunk 1 above)
Comment (aojea) [inline README.md:87]:
```suggestion
Implementing functionality that is not part of an official or proposed Kubernetes network policy API
```

File: README.md (hunk 2 above)
Comment (aojea) [reply README.md:88]: will clarify in the scope

**Pull request closed without merging**